lib/plugins/package/lib/zipService.js

```diff
@@ -102,14 +102,19 @@ module.exports = {
     );
 
     return Promise.all([ // Get file contents and stat in parallel
```

Comment:
Just a quick note here that this should possibly be `.mapSeries` rather than `.all` because `.all` does not preserve the order in which the Promises are resolved and could therefore break the array access in the `.then()`.

Reply:
@pmuens I think it's all fine with `all`. Items of arrays are resolved in indefinite order, however order of items in result array reflects order of promises (they're resolved from) in input array.

Reply:
Still I agree that we should rely here on bluebird here as well. For consistency, e.g. proper longStackTraces support.

I'll update it

Reply:
It's updated to rely on `BbPromise.all`

Reply:
> The promise's fulfillment value is an array with fulfillment values at respective positions to the original array

Ah. You're right. The array-order will be kept!

 Thanks for updating @medikoo 👍